src/ml/neural_net/tf_compute_context.cpp

```diff
@@ -212,7 +215,8 @@ float_array_image_augmenter::float_array_result tf_image_augmenter::prepare_augm
 
     // Get augmented images and annotations from tensorflow
     pybind11::object augmented_data = tf_aug.attr("get_augmented_data")(
-        data_to_augment.images, data_to_augment.annotations);
+        data_to_augment.images, data_to_augment.annotations, output_height,
+        output_width, resize_only);
```

Comment:
Can't we just pull e.g. `output_width` from `get_options`? Why do they need to be passed explicitly to this method?

Reply:
I will refactor and only pass these flags to tensorflow.